CHANGELOG.md

```diff
@@ -1,5 +1,59 @@
 ### [Versions](https://mui.com/versions/)
 
+## 5.2.5
+
+<!-- generated comparing v5.2.4..master -->
+
+_Dec 20, 2021_
+
+A big thanks to the 14 contributors who made this release possible. Here are some highlights ✨:
```

Comment:
I just merged two PRs that I think are worth highlighting 😊 

- 📓 General design refinement to the documentation (#29994)
- 🖌 Polished the design of the Base components (#30149)

Reply:
Highlight it is!